src/chatkit.ts

```diff
@@ -217,6 +218,7 @@ export interface CreateRoomOptions {
 export interface UpdateRoomOptions {
   id: string
   name?: string
+  pushNotificationTitleOverride?: string | null,
```

Comment:
the `| null` is implied by the `?`. I'm not sure if this is purely redundant or if the two have subtly different behaviour, but either way I don't think you want both

Reply:
no, the `?` means it can be `undefined`, which is not the same as `null`.

> I don't think you want both
I do. If it's explicitly set to `null`, then I want it to result in the override being removed, if it's `undefined` I want to leave it unchanged, whatever the existing value is.

Reply:
without the strict null checks option (which we don't use) they are identical. You can remove it and the semantics will be identical.

Reply:
but... maybe it's worth it for expressing intent?

Reply:
Eh, we actually do use strict null checks via the `strict` property. Ignore me. 🙃